internal/oapispec/ffi2swagger.go

```diff
@@ -35,35 +38,101 @@ func NewFFISwaggerGen() FFISwaggerGen {
 	return &ffiSwaggerGen{}
 }
 
-func (og *ffiSwaggerGen) Generate(ctx context.Context, baseURL string, ffi *fftypes.FFI) (swagger *openapi3.T, err error) {
-
-	// routes := []*Route{}
-
-	// for _, method := range ffi.Methods {
-	// 	if routes, err = og.addMethod(ctx, routes, method); err != nil {
-	// 		return nil, err
-	// 	}
-	// }
-
-	// for _, event := range ffi.Events {
-	// 	if routes, err = og.addEvent(ctx, routes, event); err != nil {
-	// 		return nil, err
-	// 	}
-	// }
-
-	return nil, nil
+func (og *ffiSwaggerGen) Generate(ctx context.Context, baseURL string, api *fftypes.ContractAPI, ffi *fftypes.FFI) (swagger *openapi3.T, err error) {
```

Comment:
Feels a bit wrong to me now that we've ended up with this in the `oapispec` package - and I know that's my fault as I was fighting with the dependencies. Doesn't need to be addressed in this PR, but I would like to do a PR that wrangles out the `oapispec` package... maybe even into a separate repo.

Reply:
Yes, I've said a few times that oapispec could be its own project, considering how much it's grown. And agreed that this code doesn't ultimately belong here.